src/components/common.js

```diff
@@ -528,10 +528,13 @@ export const HeroWrapper = ({ showMenu = true, bigSubhead = false, children }) =
         `${getAppName(true)} is a ${Utils.cond(
           [isTerra(), () => 'cloud-native platform'],
           [isFirecloud(), () => 'NCI Cloud Resource project powered by Terra'],
+          [isProjectSingular(), () => 'project funded by Additional Ventures and powered by Terra'],
           () => 'project powered by Terra'
         )} for biomedical researchers to `,
         heavyWrapper('access data'), ', ', heavyWrapper('run analysis tools'), ', ',
-        span({ style: { whiteSpace: 'nowrap' } }, ['and', heavyWrapper(' collaborate'), '.'])
+        span({ style: { whiteSpace: 'nowrap' } }, ['and', strong([' collaborate'])]),
```

Comment:
FYI, when I use VoiceOver it is breaking the text here at this span. It would be nice if it wasn't necessary to force `nowrap` around "and **collaborate**". (But don't feel like you have to keep working on this)

Reply:
> Using a non-breaking space character between them instead of wrapping in a span might work.

Good to know. Were you thinking 
```javascript
`and&nbsp;${strong([' collaborate'])}`
```
?

Reply:
> if you make the region wide enough so it can all render on one line, it doesn't split the content when reading

Do you mean by increasing `width: 575` on [line #527](https://github.com/DataBiosphere/terra-ui/pull/3134/files#diff-06c322acffe4a65c324584f6184927c7d4fca7e49530365c92671c0ea1fd30e1R527)?

Reply:
> Were you thinking
>
> `` `and&nbsp;${strong([' collaborate'])}` ``

I think React will escape `&nbsp;`, so you may have to use the Unicode character: `\u00a0`.

```js
'and\u00a0', strong(['collaborate']),
```

Reply:
To be clear, I'm not suggesting you increase the width. I was just trying it out to understand the effect of the width on how the screen reader was behaving.